ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/monitor-sessions.j2

```diff
@@ -0,0 +1,55 @@
+{% if monitor_sessions is arista.avd.defined %}
+
+## Monitor Sessions
+
+### Monitor Sessions Summary
+
```

Comment:
move this blank line down to just above the `#### {{ monitor_session.name }}`. This way it will produce a blank line for every loop, but none if the list is empty.